site/pages/api/create-disruption.test.ts

```diff
@@ -622,126 +611,4 @@ describe("create-disruption API", () => {
         );
         expect(writeHeadMock).toBeCalledWith(302, { Location: `/create-disruption/${defaultDisruptionId}` });
     });
-
-    it("should confirm that the dates are expanded as expected", () => {
```

Comment:
Why are we deleting all these tests?

Reply:
The test case was for expandedValidity function. As this function is deleted the test case would redundant